duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -256,21 +257,35 @@ public void useCustomCertificates(String[] customCaCerts) {
     httpClient = httpClient.newBuilder().certificatePinner(pinner).build();
   }
 
-  protected String canonRequest(String date, int sigVersion)
+  protected String canonRequest(String date, int inSigVersion)
       throws UnsupportedEncodingException {
+
+    int[] avaiableSigVersion = {1, 2};
+    int sigVersion = 1;
+
+    if (Arrays.stream(avaiableSigVersion).anyMatch(i -> i == inSigVersion)){
+      sigVersion = inSigVersion;
+    }
```

Comment:
I don't know if this is the `javanic` way to this. I wish I can do 

```
if sigVersion in (1,2):
```

Also fun time when java assume parameter variable is final.

Reply:
this is the technically correct approach, but it feels like massive overkill.

Reply:
I want to prevent wrong sig version as well as keep the old behavior (default to use sig v1) without throw any new errors.

Reply:
doesn't really matter though, as long as it works.